### PR TITLE
ember cli update

### PR DESCRIPTION
**Pull request by theseyi**

### CODE REVIEW

File: web/app/dao/MetricsDAO.java [hunk 1]
```diff
@@ -250,6 +250,9 @@ public static JsonNode getMetricListViewMetricNodesByMetricId(Integer metricId)
         return getMetricListViewMetricNodes(metric.dashboardName, metric.group);
     }
 
+    public final static String GET_METRIC_AUTO_COMPLETE_LIST = "SELECT DISTINCT metric_name " +
```
Comment (theseyi) [inline web/app/dao/MetricsDAO.java]: @alyiwang can you take a look at this specific commit to see if accurately reflects the current state of the MetricsDAO.java file? This was the only thin that got added in the rebase and that shouldn't be the case.

Thanks.

File: web/app/controllers/Application.java [hunk 2]
```diff
@@ -25,6 +25,11 @@
 import utils.Tree;
 import views.html.*;
 
+import com.fasterxml.jackson.databind.JsonNode;
```
Comment (theseyi) [inline web/app/controllers/Application.java]: @alyiwang Please take a look at this as well. Thanks

File: web/app/dao/MetricsDAO.java (hunk 1 above)
Comment (theseyi) [inline web/app/dao/MetricsDAO.java]: This is an error from merging. Fixed in a recent commit.

File: web/app/controllers/Application.java [hunk 3]
```diff
@@ -170,32 +175,48 @@ public static Result login()
...
-            flash("error", "Invalid username or password");
-            return redirect(controllers.routes.Application.login());
+
+        try {
+            AuthenticationManager.authenticateUser(username, password);
+        } catch (Exception e) {
+            return badRequest("Invalid credentials");
         }
+
         session().clear();
```
Comment (alyiwang) [inline web/app/controllers/Application.java]: Maybe move the "set session actions" to behind this clear(). Also need to set session "user".

Comment (theseyi) [inline web/app/controllers/Application.java]: Yes, thanks for taking a look here. Will do 👍

**Pull request merged**